tests/test_sql.py

```diff
@@ -309,3 +309,12 @@ def test_in_subquery():
         sql_fingerprint("SELECT `f1`, `f2` FROM `b` WHERE `x` IN (SELECT 1)")
         == "SELECT ... FROM `b` WHERE `x` IN (SELECT #)"
     )
+
+
+def test_insert_list():
+    assert (
+        sql_fingerprint(
+            "INSERT INTO `t` (`f1`, `f2`) VALUES (`a`, `b`), (`c`, `d`), (`c`, `d`)"
+        )
+        == "INSERT INTO `t` (...) VALUES (...), (...), (...)"
```

Comment:
ah okay, so you'd rather this was collapsed into `VALUES ...` ?

Reply:
yes

Reply:
similar to what the in clause does in the test test_in_multiple_values_and_clause

Reply:
hmmm, maybe it needs another example, you can also get stuff like
```INSERT INTO "tools_canonicalisedaddress" (...) VALUES (...[], #::double precision[], ...), (...[], #::double precision[], ...)```

Reply:
Yes please add such an example